### PR TITLE
Add Community Details

### PR DESCRIPTION
**Pull request by simonv3**
In the spirit of keeping track of who controls what and as raised in #18 and #17.

Let me know if any of these details are wrong / out of date.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,4 +11,16 @@ Want to add yourself to this list? Do the following:
...
\ No newline at end of file
+* Voila!
+
+Community
+=========
+
+Our community is spread across various services and maintained by various people. If you've got any questions about those services, please reach out to those people who can help.
+
+* GitHub (here) - [Administrators](https://github.com/orgs/opensourcedesign/teams/core)
+* Twitter - (https://twitter.com/opensrcdesign) - administered by @jancborchardt.
```
Comment (evalica) [inline README.md]: I think is better if you remove the '.' (dot)

File: README.md [hunk 2]
```diff
@@ -11,4 +11,16 @@ Want to add yourself to this list? Do the following:
...
+* Facebook - (https://www.facebook.com/opensrcdesign/) - administered by @razetime.
```
Comment (evalica) [inline README.md]: Same here for the . dot

File: README.md [hunk 3]
```diff
@@ -11,4 +11,16 @@ Want to add yourself to this list? Do the following:
...
+* Medium - (https://medium.com/open-source-design) - administered by @razetime, @plastelina, @rdbarlett
```
Comment (evalica) [inline README.md]: :) actually it needs to be tested in order for the punctuation to not interfere with the user name. Maybe there is no problem. Sorry for the noise.

File: README.md (hunk 1 above)
Comment (simonv3) [inline README.md]: You mean the period at the end of the sentence? Makes sense.

**Comment by evalica** [comment]
The idea to list contact people is very good. 

My only comments are:
1. is the list finite? how can we become part of the Administrators or have access to admin rights on Medium or Twitter, etc.? Should we set up some groups? Is this the decision of the initial owner, or a community decision? 
2. What will happen to a service if the initial owner is not active anymore? Do we kill the service account or find replacement? 

But these might be questions for the future. Anyway this is a good start. Thank for handling it.

**Comment by simonv3** [comment]
@evalica those questions is exactly why I wanted to start gathering this information together. I understand that we don't want to give just anyone access to the accounts by publishing the log-in details here, but there should be more transparency and anti-fragility to each. I'd like it if we could have at least two admin for each account.

File: README.md (hunk 1 above)
Comment (jancborchardt) [inline README.md]: Btw, @bnvk has the password as well. And anyone who would like to tweet, please ask either of us and we’ll give it to you.

File: README.md [hunk 4]
```diff
@@ -11,4 +11,16 @@ Want to add yourself to this list? Do the following:
...
-* Voila!
\ No newline at end of file
+* Voila!
+
+Community
+=========
+
+Our community is spread across various services and maintained by various people. If you've got any questions about those services, please reach out to those people who can help.
+
+* GitHub (here) - [Administrators](https://github.com/orgs/opensourcedesign/teams/core)
```
Comment (evalica) [inline README.md]: Links is not valid anymore

**Pull request merged**